### DISCUSSION TITLE
HQ LJSpeech voice. (finished, dl available)

### DISCUSSION THREAD

**Discussion by StoryHack** [rhasspy/piper#202] [Show and tell]
As I mentioned in another topic, I'm currently training a voice from scratch on the high quality setting using the LJSpeech dataset, which is in the public domain. When I'm done, I'll release the .ckpt and .onnx files into the public domain, too.

I'm not sure how long I'll end up letting it train. For a while yet, at least. The training docks say to watch for when certain graphs in the tensorboard "level off". I don't know exactly what that means, but I'm watching those graphs. I don't have a particularly spectacular GPU, so it'll take a while.

If anybody is interested, I set up a page with some results as of epoch 286 this morning. [TTS Voice Training Experiments](https://brycebeattie.com/files/tts)

**Comment by synesthesiam** [comment]
Thanks for doing this! I'd recommend running the English test phrases through as well (https://github.com/rhasspy/piper/tree/master/etc/test_sentences). The last 4 sentences are pangrams, which attempt to use every letter of the alphabet in a single sentence. They usually give a good hint of how the voice will perform on text outside the dataset.

Regarding the stopping criteria, every automated method I've found doesn't work when you have multiple optimizers. Since VITS has both the generator and discriminator loss, I'm not sure if there's a good (automated) way to say a voice is "done".

**Reply by StoryHack** [reply]
Yep, I'm using the test_en-us.jsonl file from there to generate some of the test sentences on the page I linked to, the others are the ones automatically generated by the training script and visible on the tensorboard. They do feel like a better metric, and I'm antsy enough about the whole shebang to go in and generate the sentences a couple of times a day. How many samples did the Lessac voice use to train? That one ended up being about 2200 epochs, right?

**Comment by StoryHack** [comment]
Ok, I just passed 1000 epochs, and updated[ the page with examples of what the voice is producing](https://brycebeattie.com/files/tts). Perhaps halfway there?

**Comment by StoryHack** [comment]
Ok, I finished training 2000 epochs, and here is what I ended up with. Ckpt and voice files are dedicated to the public domain. Samples are available [here](https://brycebeattie.com/files/tts).

- ckpt: [ljspeech-2000-epochs.ckpt](https://drive.google.com/file/d/1IETnZzJBpDpqhpOrsvUagsCmaQajESCT/view?usp=sharing)
- onnx [ljspeech.onnx](https://drive.google.com/file/d/1IDreJpAUq8FTj0MJuOrqOnEDvbhMPwE4/view?usp=sharing)
- config file [ljspeech.onnx.json](https://drive.google.com/file/d/1IDsDkMaDKzF8UxCzH0lel4KyUM6Kic7e/view?usp=sharing)
